src/components/limited-restoration-panel/LimitedRestorationPanel.vue

```diff
@@ -26,7 +26,7 @@
 
 <script lang="ts">
 import { Component, Emit, Mixins, Prop, Watch } from 'vue-property-decorator'
-import { DateMixin } from '@bcrs-shared-components/mixins'
+import { DateMixin } from '@/mixins' // NB: local mixin (StoryBook can't find it otherwise)
```

Comment:
Another small fix

Reply:
I don't like this one, as it means the "parent UI" needs to have this exact mixin in its local src/mixins/ folder.

I'm hoping your `moduleNameMapper` change, or something similar for SB, will allow the old code to work in all situations.